session/bootstrap.go

```diff
@@ -583,11 +583,13 @@ const (
 	version84 = 84
 	// version85 updates bindings with status 'using' in mysql.bind_info table to 'enabled' status
 	version85 = 85
+	// version86 changes global variable `tidb_enable_top_sql` value from false to true.
+	version86 = 86
 )
 
 // currentBootstrapVersion is defined as a variable, so we can modify its value for testing.
 // please make sure this is the largest version
-var currentBootstrapVersion int64 = version84
+var currentBootstrapVersion int64 = version86
```

Comment:
emm. where is the version85

Reply:
here: https://github.com/pingcap/tidb/pull/33184